website/docs/ecosystem/mem0.md

```diff
@@ -1,4 +1,4 @@
-# Mem0: Empower your AI applications with long-term memory and personalization
+# Mem0: Agent's Long-term Memory and Personalization
```

Comment:
This read a little awkward @Dev-Khant, perhaps consider this alt:
```suggestion
# Mem0: Long-Term Memory and Personalization for Agents
```

Reply:
Thanks for the suggestion!